src/app/containers/App/App.jsx

```diff
@@ -58,15 +69,20 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
         errorCode: null,
       });
 
-      route.getInitialData(location.pathname).then(data =>
-        setState(prevState => ({
-          ...prevState,
-          loading: false,
-          pageData: path(['pageData'], data),
-          status: path(['status'], data),
-          error: path(['error'], data),
-        })),
-      );
+      route
+        .getInitialData(location.pathname)
+        .then(data =>
+          setState(prevState => ({
+            ...prevState,
+            loading: false,
+            pageData: path(['pageData'], data),
+            status: path(['status'], data),
+            error: path(['error'], data),
+          })),
+        )
+        .then(() => {
```

Comment:
can we remove the `.then` here and move the `shouldSetFocus.current = true;` into the previous then before the `setState`? I'm wondering if this is causing the issues I've been describing. I wouldn't have thought so though.